lib/elixir/lib/enum.ex

```diff
@@ -1761,7 +1761,7 @@ defmodule Enum do
 
   """
   @doc since: "1.4.0"
-  @spec split_with(t, (element -> any)) :: {list, list}
+  @spec split_with(t, (element -> as_boolean(term))) :: {list, list}
```

Comment:
or should it be `as_boolean(element)` ?

Reply:
Not really since the return value may have no relationship with the element.